FAQs/README.md

```diff
@@ -70,6 +71,28 @@ To update the telemetry preferences, go to resource group where AzTS solution ha
   - If AIConfigurations\_\_AnonymousUsageTelemetry\_\_LogLevel is either 'All' or 'Onboarding', then specify the team's contact DL in this.
   - If AIConfigurations\_\_AnonymousUsageTelemetry\_\_LogLevel is either 'Anonymous' or 'None', then specify 'N/A' in this.
   
+### **5. On running the Autoupdater function getting some error related to function runtime version.**
+
+
+- Run below command for AzSK-AzTS-AutoUpdater-xxxxx function app. 
+
+  ``` Powershell
+  $appSettings = @{}
+  $appSettings.add("FUNCTIONS_EXTENSION_VERSION","~4")
+  $appSettings.add("WEBSITE_RUN_FROM_PACKAGE","https://aka.ms/DevOpsKit/AzTS/V4/AutoUpdater")
+
+  Update-AzFunctionAppSetting -AppSetting $appSettings -Name <APP_NAME> -ResourceGroupName <RESOURCE_GROUP_NAME> -Force
+
+  Set-AzWebApp -NetFrameworkVersion v6.0 -Name <APP_NAME> -ResourceGroupName <RESOURCE_GROUP_NAME>
+  ```
+- Run below command for AzSK-AzTS-MetadataAggregator-xxxxx and AzSK-AzTS-WorkItemProcessor-xxxxx.
+
+  ``` Powershell
+  Update-AzFunctionAppSetting -AppSetting @{"FUNCTIONS_EXTENSION_VERSION" = "~4"} -Name <APP_NAME> -ResourceGroupName <RESOURCE_GROUP_NAME> -Force
+
+  Set-AzWebApp -NetFrameworkVersion v6.0 -Name <APP_NAME> -ResourceGroupName <RESOURCE_GROUP_NAME>
+  ```
+- Try running the Autoupdater once again.
```

Comment:
We need not start with "Try running..."

Step 3: Re-rerun the Autoupdater after completing Step 2 and the operation should complete successfully. This also ensures that the ..... (whatever the function run time was supposed to get updated to).